### DISCUSSION TITLE
RandCropByPosNegLabeld() can't support two labels

### DISCUSSION THREAD

**Discussion by vectorzwt** [Project-MONAI/tutorials#1651] [General]
My task requires segmenting images from two time periods simultaneously, so my model needs to accept two inputs at the same time and get two results. When building a data set, I want to use the 'RandCropByPosNegLabeld' function to get the patch. Are there any good solutions to this problem?

**Comment by KumoLiu** [comment]
Hi @vectorzwt, I understand you're looking to use the `RandCropByPosNegLabeld` function. If your goal is to generate two separate inputs based on different labels, calling the function twice, once for each label, should suffice. Please let me know if this aligns with your intended use-case or if you need further assistance.
Thanks.
